aiida_vasp/io/poscar.py

```diff
@@ -78,3 +85,37 @@ def poscar_str(self):
     def write(self, path):
         destination = py_path.local(path)
         destination.write(self.poscar_str())
+
+
+class PoscarParser(BaseFileParser):
```

Comment:
Unit tests to aiida_vasp/io/tests/test_poscar_io.py, please

Reply:
Added test_poscar_io.py and a reference POSCAR file.